dashboard/test/logic/task_grid_filter_test.dart

```diff
@@ -140,19 +140,19 @@ void main() {
     expect(falseFilter, isNot(equals(trueFilterMap)));
     expect(falseFilter, isNot(equals(trueFilter)));
 
-    expect(trueFilter.matchesTask(QualifiedTask.fromTask(Task()..stageName = stageName)), true);
-    expect(trueFilterMap.matchesTask(QualifiedTask.fromTask(Task()..stageName = stageName)), true);
+    expect(trueFilter.matchesTask(QualifiedTask.fromTask(Task()..builderName = taskName)), true);
+    expect(trueFilterMap.matchesTask(QualifiedTask.fromTask(Task()..builderName = taskName)), true);
 
-    expect(falseFilter.matchesTask(QualifiedTask.fromTask(Task()..stageName = stageName)), false);
-    expect(falseFilterMap.matchesTask(QualifiedTask.fromTask(Task()..stageName = stageName)), false);
+    expect(falseFilter.matchesTask(QualifiedTask.fromTask(Task()..builderName = taskName)), false);
+    expect(falseFilterMap.matchesTask(QualifiedTask.fromTask(Task()..builderName = taskName)), false);
   }
 
-  test('matches Luci stage', () {
+  test('matches Windows stage', () {
```

Comment:
For tests like this, I prefer writing them with a table driven design. Something like:

```dart
for (String platform in platforms) {
  test('matches $platform', () {
    testStage(...); // This gets somewhat tricky, and you likely need to do make [platform] a map to have all the values
  });
}
```

Reply:
Done but I am not happy how I am setting the taskgridfilter, as it is the same way inside testStage. What do you think?

Reply:
I'll leave it, since you approved the PR and the test is functional, just weird looking :)